<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="urlLabel">
          <property name="text">
           <string>输入地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="urlLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="forwardPushButton">
          <property name="text">
           <string>前进</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backPushButton">
          <property name="text">
           <string>后退</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopPushButton">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reloadPushButton">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWebView" name="webView">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_K">
    <property name="title">
     <string>查看(&amp;K)</string>
    </property>
    <addaction name="viewCodeAction"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="setDifferentAction"/>
   </widget>
   <addaction name="menu_K"/>
   <addaction name="menu_S"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="viewCodeAction"/>
   <addaction name="setDifferentAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="setDifferentAction">
   <property name="text">
    <string>设置主页(&amp;Y)</string>
   </property>
  </action>
  <action name="viewCodeAction">
   <property name="text">
    <string>查看源代码(&amp;S)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>forwardPushButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backPushButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reloadPushButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopPushButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
